### DISCUSSION TITLE
bug sonar

### DISCUSSION THREAD

**Discussion by sakif-nabil** [lzybkr/PSReadLine#3316] [Q&A]
![bugSonar](https://user-images.githubusercontent.com/53219441/168786100-e6c91a83-2663-4787-9a3e-7c4d37c2b94c.PNG)

**Comment by daxian-dbw** [comment]
This issue was already fixed (see #1306). Please upgrade to the [2.2.3 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.2.3) from PowerShell Gallery.
See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions. Please let us know if you run into the same issue with the latest version.
